plugins/MauticClearbitBundle/Form/Type/BatchLookupType.php

```diff
@@ -39,7 +42,7 @@ public function buildForm(FormBuilderInterface $builder, array $options)
 
         $builder->add(
             'notify',
-            'yesno_button_group',
+            YesNoButtonGroupType::class,
```

Comment:
This will probably cause conflicts with Alan's refactoring of the CoreBundle. But it won't be hard to deal with it. I replace only the Symfony field types.